ocp_resources/virtual_machine_instance.py

```diff
@@ -102,6 +103,13 @@ def virt_handler_pod(self):
 
         raise ResourceNotFoundError
 
+    @property
+    def vm(self):
+        vms = list(VirtualMachine.get(name=self.name, namespace=self.namespace))
```

Comment:
```
vm = VirtualMachine(name=self.name, namespace=self.namespace)
if vm.exists:
    return vm
raise ResourceNotFoundError(.......error......)
````

Reply:
raise is not needed as vmi can exist without a vm

Reply:
get if from the vmi
metadata.ownerReferences.....

```
  ownerReferences:
  - apiVersion: kubevirt.io/v1
    blockOwnerDeletion: true
    controller: true
    kind: VirtualMachine
    name: fedora-relieved-beetle
    uid: c7435302-d1da-4d78-abfd-d9b7ac384544

```

Reply:
please remove (and also the log about VM below)

Reply:
removed